include/rocksdb/options.h

```diff
@@ -221,6 +221,10 @@ struct ColumnFamilyOptions : public AdvancedColumnFamilyOptions {
   // Default: kDisableCompressionOption (Disabled)
   CompressionType bottommost_compression = kDisableCompressionOption;
 
+  // different options for compression algorithms used by bottommost_compression
+  // if it is enabled
```

Comment:
can we be more explicit about what does "enabled" mean here? Maybe point the user to the `CompressionOpts::enabled` comment and thoroughly document it there.

Reply:
I think it will be better to point out that if they enable it and what to specify the options for bottommost compression, they should look at the bottommost_compression_opts and CompressionOptions definition.